src/static/js/ace.js

```diff
@@ -241,7 +241,7 @@ const Ace2Editor = function () {
     innerFrame.allowTransparency = true; // for IE
     // The iframe MUST have a src or srcdoc property to avoid browser quirks. See the comment above
     // outerFrame.srcdoc.
-    innerFrame.srcdoc = '<!DOCTYPE html>';
+    innerFrame.src = '/aceInner.html';
```

Comment:
```suggestion
    innerFrame.src = '../aceInner.html';
```

Reply:
I actually needed `innerFrame.src = 'aceInner.html';` for it to work for me,

Reply:
Oops, yup, you're right.